### PR TITLE
Disable 2 configuration tests failing on unix

### PR DESCRIPTION
**Pull request by danmosemsft**
related to https://github.com/dotnet/corefx/issues/15065
related to https://github.com/dotnet/corefx/issues/15066

### CODE REVIEW

**Comment by mmitche** [comment]
@diotnet-bot test portable linux debug pipeline
@diotnet-bot test portable linux release pipeline

**Comment by stephentoub** [comment]
cc: @JeremyKuhne

**Comment by danmosemsft** [comment]
The Unix failures are all being addressed by other pr's. This can be merged.

File: src/System.Configuration.ConfigurationManager/tests/Mono/ConfigurationManagerTest.cs [hunk 1]
```diff
@@ -67,6 +67,7 @@ public void OpenExeConfiguration1_UserLevel_PerUserRoaming()
         }
 
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)] // https://github.com/dotnet/corefx/issues/15065
```
Comment (stephentoub) [inline src/System.Configuration.ConfigurationManager/tests/Mono/ConfigurationManagerTest.cs]: Shouldn't these be ActiveIssue rather than PlatformSpecific?

Comment (danmosemsft) [inline src/System.Configuration.ConfigurationManager/tests/Mono/ConfigurationManagerTest.cs]: Yes I forgot Santi's added platform to ActiveIssue. Will fix

Comment (stephentoub) [inline src/System.Configuration.ConfigurationManager/tests/Mono/ConfigurationManagerTest.cs]: FWIW, TestPlatforms has been part of ActiveIssue for a long time.

**Pull request merged**